Translators/Incident/pgquery.py

```diff
@@ -0,0 +1,86 @@
+import os
```

Comment:
Is there any of this file we could reuse or abstract to a library and use amongst any of our translators that need it?

Reply:
I think we can abstract all of this functionality to a library as this file is the same for each translator.

Reply:
I've opened the PR for abstracting the shared functionality in #24